.circleci/config.yml

```diff
@@ -24,6 +24,9 @@ aliases:
         command: |
           /bin/bash -c 'echo -e "[mysqld]\ndatadir = /dev/shm" > /etc/my.cnf.d/obs.cnf && cp -a /var/lib/mysql/* /dev/shm && /usr/lib/mysql/mysql-systemd-helper start'
         name: db
+      - image: registry.opensuse.org/obs/server/unstable/container/sle12/sp3/images/openbuildservice/frontend-backend:latest
```

Comment:
Why the frontend-backend image and not only the backend?

Reply:
I think we already discussed the issue: You don't have a git checkout in the secondary containers, therefore this will fail. You need to switch to docker-compose or change the main image to frontend-backend and run the backend on the same machine (like we do in minitest, although we start the test backend in minitest and not the development backend)

Reply:
we can't have that because then you the backend server would be localhost for *some* rspec. Switching to docker-compose is the only option left - alreay came to the same conclusion with selenium :(